spec/mailers/jobapps_mailer_spec.rb

```diff
@@ -145,6 +145,39 @@
     end
   end
 
+  describe 'send_note_for_later' do
+    let(:record) do
+      create :application_record,
+             saved_for_later: true,
+             note_for_later: 'We need you to grow up a little'
+    end
+    let!(:template) do
```

Comment:
I needed to create a template so I had an email. The factory doesn't have one by default because it is not a required attribute.